src/PSR7Session/Storage/StorageInterface.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace PSR7Session\Storage;
+
+use PSR7Session\Id\SessionIdInterface;
+use PSR7Session\Session\SessionInterface;
+
+interface StorageInterface
```

Comment:
I would suggest starting an example implementation (file-based) to see what the problems/missing parameters are.

One that I can think of is surely the session expiration.